NEWS.md

```diff
@@ -1,5 +1,39 @@
 # NEWS
 
+## 1.0.1 - 2025-02-21
+
+### Package distribution update
+
+In previous releases, Ubuntu packages were available only through
+https://launchpad.net/~groonga/+archive/ubuntu/ppa. Starting with this release,
```

Comment:
https://groonga.org/docs/install/ubuntu.html#ppa-personal-package-archive said "PPA on Launchpad". In general, users don't see https://launchpad.net/~groonga/+archive/ubuntu/ppa URL when they use. Because they use `add-apt-repository ppa:groonga/ppa`.

BTW, it said that "The Groonga APT repository for Ubuntu"...